addons/mrp/models/stock_orderpoint.py

```diff
@@ -124,3 +124,7 @@ def _post_process_scheduler(self):
             ('state', '=', 'draft'),
         ]).action_confirm()
         return super()._post_process_scheduler()
+
+    def _is_allowed_product(self, product):
+        # don't create an order point for kit products
+        return not product.bom_ids.filtered(lambda x: x.type == 'phantom')
```

Comment:
I'm a little bit worried about the performance of this as their is no `bom_find` multi in 14.0
Instead, I would do a gather of all kit boms -> get their products and exclude them in the read_group

Reply:
that's what I did initially but it caused an issue when a product has multiple variants where some are kits and other aren't
https://github.com/odoo/odoo/pull/107231#issuecomment-1375715780